source/parser.hera

```diff
@@ -6715,7 +6715,7 @@ NestedEnumProperties
     }
 
 NestedEnumProperty
-  Nested EnumProperty ->
+  Nested? EnumProperty _? ->
```

Comment:
I'd do this slightly differently, similar to how we do lines of array items (see https://github.com/DanielXMoore/Civet/blob/d2a6b8b9e3e8b4d42b4f1bef8a4c2350625572b8/source/parser.hera#L2670)

Reply:
I've been trying to figure out whether they are equivalent. One issue with the current code is that it does not enforce a leading `Nested`. But I think because of the prior `PushIndent`, the first `NestedEnumProperty` must in fact have a leading `Nested`. And `Nested` is the only way to get to the next line.

That said, this is a lot of thought, so maybe it is better to require `Nested` and explicitly consider a full line? Something like `Nested EnumProperty ( _? EnumProperty )*` might suffice.

Reply:
I did some testing on the CLI to make sure that none of these were accepted:
```civet
enum E1 A

enum E2/**/A

enum E3
A
```
I didn't add them as test cases because I wasn't sure how, but now that I've seen 518d041556f60d5fad2244691db07294b25689e2 I can add these three as negative test cases.

Reply:
I agree that none of them work, but it takes a lot of thought to convince ourselves that it's correct, so the more explicit rule seems better?